app/styles/_icons.scss

```diff
@@ -252,10 +252,15 @@ i.fa-circle-o-notch.fa-spin {
 }
 
 .icon--search {
-  width: 21px;
-  height: 21px;
+  width: 20px;
+  height: 23px;  
 }
 
+.searchbox--slim .icon--search {
+  width: 14px;
+  height: 15px;
+} 
+
```

Comment:
Cool. Maybe we can talk tomorrow after the standup. I don't think I'll have time to work on it today.